crates/bevy_core_pipeline/src/tonemapping/tonemapping_shared.wgsl

```diff
@@ -1,5 +1,23 @@
 #define_import_path bevy_core_pipeline::tonemapping
 
+#from bevy_render::view import View, ColorGrading
+
+// hack !! not sure what to do with this
+#ifdef TONEMAPPING_PASS
+    @group(0) @binding(3)
+    var dt_lut_texture: texture_3d<f32>;
+    @group(0) @binding(4)
+    var dt_lut_sampler: sampler;
+#else
+    @group(0) @binding(14)
+    var dt_lut_texture: texture_3d<f32>;
+    @group(0) @binding(15)
+    var dt_lut_sampler: sampler;
+#endif
+// more hack! hope it's always there...
```

Comment:
The code already assumes it is bound, though it doesn't specify where other than in a variable named `view`.

Reply:
oops, removed. i later realised the view it was only used for color grading which is only 4 f32s, so decided it was better to pass that in explicitly where required.